Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -204,7 +204,7 @@ bool _CloneDWriteGlyphRun(_In_ DWRITE_GLYPH_RUN const* src, _Out_ DWRITE_GLYPH_R
     for (size_t i = 0; i < [subString length]; i += attributeRange.length) {
         NSDictionary* attribs = [static_cast<NSAttributedString*>(string) attributesAtIndex:i + range.location
                                                                       longestEffectiveRange:&attributeRange
-                                                                                    inRange:{ i, [subString length] }];
+                                                                                    inRange:{ i + range.location, [subString length] }];
```

Comment:
>inRange:{ i + range.location, [subString length] }]; [](start = 84, length = 52)

Mm. This is kind of a big ask, but in retrospect, I would prefer if you relied on [NSAttributedString enumerateAttributesInRange:options:usingBlock:] to do the loop instead. You're a lot less likely to make a mistake that way. (Note that you'd have to declare incompatibleAttributeFlag and textLayout as __block if you do this)